addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -116,6 +116,7 @@ options.registry.gallery = options.Class.extend({
 
                 }
             },
+            imagesOptimizationDisabled: this.options.wysiwyg.imagesOptimizationDisabled,
```

Comment:
Why isn't the endpoint, here the `MediaDialogWrapper` in charge of doing that?
This is not robust, anyone who would call the `MediaDialogWrapper` will have to think about that, they wont (and shouldn't have to).
The `MediaDialogWrapper` (or even lower level endpoint, didn't check), should be in charge of retrieving that value when called/created so any caller will have a correctly instantiated Media Dialog with the correct disabled optimization option.

Having to patch the `s_image_gallery` is a clear hint that something was made overcomplicated